.travis.yml

```diff
@@ -1,8 +1,8 @@
+dist: bionic
 sudo: false
 language: python
 python:
-    - "3.6"
-    - "2.7"
+    - "3.8"
```

Comment:
3.6 doesn't work?

Reply:
It might, but quit living in the past!

Reply:
Lol, but the CI/CD must reflect the production platform (3.6) ;-)